app/include/zmk/keymap_hid.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#pragma once
+
+#include <usb/usb_device.h>
+#include <usb/class/usb_hid.h>
+
+#include <zmk/keys.h>
+#include <dt-bindings/zmk/hid_usage.h>
+#include <dt-bindings/zmk/hid_usage_pages.h>
+
+#ifndef HID_USAGE_PAGE16
+#define HID_USAGE_PAGE16(page, page2)                                                              \
+    HID_ITEM(HID_ITEM_TAG_USAGE_PAGE, HID_ITEM_TYPE_GLOBAL, 2), page, page2
+#endif
+
+#ifndef HID_USAGE6
+#define HID_USAGE16(page, page2) HID_ITEM(HID_ITEM_TAG_USAGE, HID_ITEM_TYPE_LOCAL, 2), page, page2
+#endif
+
+#define KEYMAP_HID_MAX_BYTES 64
+
+static const uint8_t zmk_keymap_hid_report_desc[] = {
```

Comment:
This will create a separate instance of this constant for every .c file that includes this header. Is this intended?

Reply:
This wasn't intended to be added. Some experimental file I accidentally added.